smart_control/utils/conversion_utils.py

```diff
@@ -23,12 +23,12 @@
 import types
 from typing import Mapping, Tuple
 
+from google.protobuf import timestamp_pb2
 import holidays
 import numpy as np
 import pandas as pd
-from smart_control.proto import smart_control_reward_pb2
 
-from google.protobuf import timestamp_pb2
```

Comment:
Do we want to treat `google.protobuf` as a normal third-party package, or separate it under the `smart_control` local module imports?